app/src/display/Kconfig

```diff
@@ -6,16 +6,22 @@ menuconfig ZMK_DISPLAY
     default n
     select DISPLAY
     select LVGL
-    select LV_THEMES
-    select LV_THEME_MONO
```

Comment:
I haven't been able to find where these two settings are defined in Zephyr/LVGL or through `menuconfig`; everything seems to work fine for me without them so I added a commit removing these. If anyone thinks this is necessary I can drop that specific commit from the PR.

Reply:
I can't either... Odd it works. Fine to remove, IMHO.